sequencer.c

```diff
@@ -4635,9 +4635,13 @@ static int make_script_with_merges(struct pretty_print_context *pp,
 		else
 			strbuf_addbuf(&label, &oneline);
 
+		/* sanitize label */
 		for (p1 = label.buf; *p1; p1++)
 			if (isspace(*p1))
 				*(char *)p1 = '-';
+			/* check if non alphanumeric ascii character */
+			else if (!(*p1 & 0x80) && !isalnum(*p1))
+				*(char *)p1 = '-';
```

Comment:
As you pointed out in a follow-up PR comment, these two `if` conditions should be consolidated, as the conditional code is the _exact_ same in both cases: replace the current character by a dash.